lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction.js

```diff
@@ -0,0 +1,138 @@
+const clone = require('../../util/clone');
```

Comment:
I deleted the empty file -- I think Anthony committed it by accident when he started working on this task a little while ago, haha.